task_5_2.py

```diff
@@ -0,0 +1,15 @@
+"""Introduction to programming’: Task 2,
+   Kyshynska Taisiia"""
+
+print("""Introduction to programming’: Task 2
+         Kyshynska Taisiia""")
+
+n = int(input("Введіть ціле число: "))
+
+k = 0  # лічильник цифр
+
+while n != 0:
```

Comment:
```suggestion
while n:
```

Reply:
another example for consideration:

from itertools import takewhile, count, repeat

row = ((i, n) for i, n in enumerate(repeat(n)))
print(len(list(takewhile(lambda x: x[1] // (10 ** x[0]) > 0, row))))